scripts/hypothesistooling.py

```diff
@@ -79,3 +89,27 @@ def create_tag():
     git("config", "core.sshCommand", "ssh -i deploy_key")
     git("tag", __version__)
     git("push", "--tags")
+
+
+def build_jobs():
+    """Query the Travis API to find out what the state of the other build jobs
+    is. Note: This usage of Travis has been somewhat reverse engineered due to
+    a certain dearth of documentation as to what values what takes when."""
+    import requests
+
+    build_id = os.environ["TRAVIS_BUILD_ID"]
+
+    url = "https://api.travis-ci.org/builds/%s" % (build_id,)
+    data = requests.get(url, headers={
+        'Accept': "application/vnd.travis-ci.2+json"
+    }).json()
+
+    matrix = data["jobs"]
+
+    jobs = {}
+
+    for m in matrix:
+        name = m["config"]["env"].replace("TASK=", "")
+        status = m["state"]
+        jobs.setdefault(status, []).append(name)
```

Comment:
I actually prefer not to do that because then the fact that it's a defaultdict leaks out into the caller and potentially introduces issues when that's unexpected.

Not that it really matters either way here.